app/controllers/manage_evaluators_controller.rb

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+class ManageEvaluatorsController < ApplicationController
+  include ManageEvaluatorsHelper
+
+  before_action :set_challenge
+
+  VALID_EVALUATOR_ROLES = %w[evaluator solver challenge_manager].freeze
+
+  def index
+    @phases = @challenge.phases.order(:start_date)
```

Comment:
this could be moved into a `before_action` (I usually do data loading tasks there)
```
before_action :set_phases, only: [:index]
```